app/views/articles/show.html.haml

```diff
@@ -6,6 +6,31 @@
   = t('author') 
   = @article.user.first_name
 %p= @article.created_at.strftime("%F")
+ 
+%h2 Comments
+- @article.comments.each do |comment|
+  %p
+    %strong Commenter:
+    = comment.user.first_name
+    = comment.user.last_name
+  %p
+    %strong Comment:
+    = comment.body
+
+-if user_signed_in?
```

Comment:
Are all users supposed to be allowed to comment? The PR title suggests that onlyt Subscribers are allowed to do that.

Reply:
we have talked through the group and have decided that we should allow all users to comment.